site/docs/types/1/0115-teams.md

```diff
@@ -0,0 +1,12 @@
+<!-- SPDX-License-Identifier: CC-BY-4.0 -->
+<!-- Copyright Contributors to the Egeria project. -->
+
+# 0115 Teams
+
+A team is a group of people who are working to a common goal. The [ActorProfile](0110-actors.md) is extended to capture more information about a team. This is recorded in the Team entity.
+
```

Comment:
- Should have a `## TeamLeader` heading
- Italicised words should not have spaces, since they refer to type names (that do not have spaces)
- PersonRole should be back-ticked as `PersonRole`
- Link should be directly to the heading (i.e. `/egeria-docs/types/1/0112-people/#personrole`)

Ideally we should also have a separate `## TeamMember` heading, and add some words to distinguish between team leaders and team members in the descriptions beneath each heading. (It's probably obvious to us, but worth being precise.)